qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/UserAccountSteps.java

```diff
@@ -28,6 +29,9 @@
 public class UserAccountSteps extends Steps {
 
 	private static final String SYSTEM_ALERT = "Atleast 8 character(s) are required";
+	private static final String CLERK = RandomStringUtils.randomAlphabetic(8);
```

Comment:
How does this attribute work in specific terms. how does it generate different username

Reply:
For each run RandomStringUtils.randomAlphabetic() generates a unique String that is passed when setting the [username](https://github.com/openmrs/openmrs-contrib-qaframework/blob/5bb1758a8737c188e79e14f2250fe554b042026d/qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/UserAccountSteps.java#L83)